variables.ini.dist

```diff
@@ -95,7 +95,7 @@ SUPPORT_EMAIL_SIGNATURE = # CyVerse Atmosphere Team
 SUPPORT_LINKS = {} # {'getting_started': "https://link_to.wiki.org/Using+Instances", 'new_provider': "https://link_to.wiki.org/Changing+Providers", "faq": "https://link_to.wiki.org/Cloud_FAQ"}
 REPLICATION_PROVIDER_LOCATION = # iPlant Cloud - Tucson
 MONTHLY_RESET_PROVIDER_LOCATIONS = [] # ['iPlant Cloud - Tucson']
-USER_EMAIL_LOOKUP_METHOD = # ldap_get_email_info
+USER_EMAIL_LOOKUP_METHOD = "django_get_email_info" # If ldap, ldap_get_email_info
```

Comment:
When we run tests we generate variables.ini from this dist file. It's really just a mistake to have all these variables be empty.